src/utp_stream.cpp

```diff
@@ -44,9 +44,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/cstdint.hpp>
 #include <limits>
 
-#define TORRENT_UTP_LOG 0
-#define TORRENT_VERBOSE_UTP_LOG 0
```

Comment:
as far as I can tell, there will not be any verbose logging done with this patch. look for places where this define is used